xml/dhcp_vendor_class.xml

```diff
@@ -8,7 +8,7 @@
     and inform (8) messages.
   -->
 
-  <fingerprint pattern="^[Mm]fg=(?:Fuji)?(?i:Xerox);[Tt]yp=(?:MFP|AIO|[Pp]rinter);[Mm]od=(?:Xerox )?(\S+) ([a-zA-Z0-9]+).*;[Ss]er=([A-Z0-9]{9,10})(?:;[Ll]oc=.*)?$">
+  <fingerprint pattern="(?i)^[Mm]fg=(?:Fuji)?Xerox;[Tt]yp=(?:MFP|AIO|[Pp]rinter);[Mm]od=(?:Xerox )?(\S+) ([a-zA-Z0-9]+).*;[Ss]er=([A-Z0-9]{9,10})(?:;[Ll]oc=.*)?$">
```

Comment:
With case insensitivity applied to the whole pattern the explicit case matching is no longer necessary.
* `[Mm]`
* `[Tt]`
* `[Pp]`
* `[Ss]`
* `[Ll]`

Reply:
Good catch, fixed.